auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -155,6 +155,23 @@
               <span>This filing will become effective on {{ formatDate(item.effectiveDate) }}</span>
             </v-tooltip>
           </template>
+          <!-- Alert Icon for CHANGE_REQUESTED status -->
+          <template v-if="status(item) === BusinessState.CHANGE_REQUESTED">
+            <v-tooltip bottom>
+              <template #activator="{ on, attrs }">
+                <v-icon
+                  class="ml-1"
+                  color="#F8661A"
+                  v-bind="attrs"
+                  v-on="on"
+                >
+                  mdi-alert
+                </v-icon>
+              </template>
+              <span> {{ (item.corpType?.code || item.corpType) === CorpTypes.REGISTRATION ? 'This registration': 'This application' }}
+                requires you to make changes before resubmitting.</span>
```

Comment:
Where did the requirement for "registration" come from? Right now this status is only for continuation applications.

Reply:
Thanks, corrected